conduit/data/datasets/audio/ecoacoustics.py

```diff
@@ -111,13 +120,18 @@ def __init__(
             self._extract_metadata()
 
         self.metadata = pd.read_csv(self.base_dir / self._METADATA_FILENAME)
+        # map numerical indices to target attributes
+        self.concept_label_decoder = self.metadata[self.md_attrs].to_dict()
```

Comment:
```suggestion
        self.md_decoder = self.metadata[self.md_attrs].to_dict()
```
Doesn't this need to be paired with the label encoded data?

Reply:
indeed :joy: